arch/x86/deflate_quick.c

```diff
@@ -122,7 +124,7 @@ static inline long compare258(const unsigned char *const src0, const unsigned ch
 }
 
 static const unsigned quick_len_codes[MAX_MATCH-MIN_MATCH+1];
-static const unsigned quick_dist_codes[8192];
+static const unsigned quick_dist_codes[QUICK_WINDOW_SIZE];
```

Comment:
Completely unnecessary change...

Reply:
It seems proper that `quick_len_codes` depends on #define value so `quick_dist_codes` should as well. I had to create the #define for checking bounds, might as well use it where can.